graph/telemetry/istio/appender/health_config.go

```diff
@@ -0,0 +1,52 @@
+package appender
+
+import (
+	"github.com/kiali/kiali/graph"
+	"github.com/kiali/kiali/models/annotation"
+)
+
+const HealthConfigAppenderName = "healthConfig"
+
+// Name: annotation
+type HealthConfigAppender struct{}
+
+// Name implements Appender
+func (a HealthConfigAppender) Name() string {
+	return HealthConfigAppenderName
+}
+
+// AppendGraph implements Appender
+func (a HealthConfigAppender) AppendGraph(trafficMap graph.TrafficMap, globalInfo *graph.AppenderGlobalInfo, namespaceInfo *graph.AppenderNamespaceInfo) {
+	if len(trafficMap) == 0 {
+		return
+	}
+
+	if getWorkloadList(namespaceInfo) == nil {
+		sdl, err := globalInfo.Business.Svc.GetServiceDefinitionList(namespaceInfo.Namespace)
+		graph.CheckError(err)
+		namespaceInfo.Vendor[workloadListKey] = &sdl
+	}
+
```

Comment:
Still missing the getServiceDefinitionList.  See my previous review comment.

Reply:
I added it in L25 @jshaughn 

https://github.com/kiali/kiali/blob/a542e95b525a6e0f903d6f236e333c818d88cc87/graph/telemetry/istio/appender/health_config.go#L25-L27